app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -75,19 +80,66 @@ class SchemeMembersListController @Inject()(
       form(manualOrUpload, membersDetails.size >= maxSchemeMembers)
         .bindFromRequest()
         .fold(
-          formWithErrors =>
-            BadRequest(
-              view(formWithErrors, viewModel(srn, page, manualOrUpload, mode, membersDetails.map(_.fullName)))
-            ),
+          formWithErrors => {
+            filterDeletedMembers(srn).map { filteredMembers =>
```

Comment:
Looks this is breaking the display of the validation errors, i.e. if you don't select a value on the radio button.

Reply:
fixed